data_prep/project_context/context_introspector.py

```diff
@@ -283,46 +69,35 @@ def _get_files_to_include(self) -> list[str]:
     for current_type in types:
       info_list = introspector.query_introspector_type_info(
           self._benchmark.project, current_type)
-      print("INFO LIST: {}".format(info_list))
       if not info_list:
+        logging.debug('Could not retrieve info for type: %s', current_type)
         continue
 
       for info in info_list:
-        source_file = self._get_source_file(info)
-        source_file = os.path.normpath(source_file)
-        source_base = os.path.basename(source_file)
+        include_file = self._get_source_file(info)
+        include_file = os.path.normpath(include_file)
+        include_base = os.path.basename(include_file)
 
-        # Ensure source_base is a file.
-        if not source_base or '.' not in source_base:
-          logging.debug('File %s found as a source path', source_file)
+        # Ensure include_file is a file.
+        if not include_base or '.' not in include_base:
+          logging.debug('File %s found as a source path', include_file)
```

Comment:
Same as above:
1. Could we log the project name to assist in debugging?
2. Please use `warning` to surface this.